enterprise/api/v1/serializers.py

```diff
@@ -660,7 +660,7 @@ class EnterpriseGroupMembershipSerializer(serializers.ModelSerializer):
     )
     enterprise_group_membership_uuid = serializers.UUIDField(source='uuid', allow_null=True, read_only=True)
     activated_at = serializers.DateTimeField(required=False)
-
+    name = serializers.CharField(source='group.name')
```

Comment:
[nit] Should we make it `group_name` in this instance?  The endpoint is exposing the concept of a 'membership' that ties a user to a group, and without context someone might assume 'name' refers to the user rather than the group.